docs/contributor/04-30-ci-cd.md

```diff
@@ -0,0 +1,50 @@
+| [`Pull Request / lint`](https://github.com/kyma-project/istio/blob/main/.github/workflows/pull-request.yaml#L24)                                                        | Responsible for linting, static code analysis.                                                                                                  |
+| [`Pull Request / Run unit tests`](https://github.com/kyma-project/istio/blob/main/.github/workflows/pull-request.yaml#L39)                                              | Responsible for running unit tests with code coverage information.                                                                              |
+
+## Pipelines running on main branch
+
+The following CI jobs run on main branch. They generate production ready artifacts and run integration tests against them to verify functional correctness of Istio Manager including possibility of upgrade to the new version.
+
+| Name                                                                                                                                                      | Description                                                                                                                                 |
+|-----------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------|
+| [`post-istio-operator-build`](https://github.com/kyma-project/test-infra/blob/main/prow/jobs/istio/istio-manager.yaml#L308)                               | Builds Istio operator image and pushes it into the prod registry.                                                                           |
+| [`post-istio-module-build`](https://github.com/kyma-project/test-infra/blob/main/prow/jobs/istio/istio-manager.yaml#L163)                                 | Builds ModuleTemplate for the Istio module and pushes it to the prod registry.                                                              |
+| [`post-istio-integration`](https://github.com/kyma-project/test-infra/blob/main/prow/jobs/istio/istio-integration.yaml#L87)                               | Executes main integration test suite for Istio Manager on k3d cluster.                                                                      |
+| [`main integration tests / Wait for image build`](https://github.com/kyma-project/istio/blob/main/.github/workflows/main-integration.yaml#L20)            | Waits for operator image build to finish so the image dependent jobs can be executed.                                                       |
```

Comment:
```suggestion
| [`main integration tests / Wait for image build`](https://github.com/kyma-project/istio/blob/main/.github/workflows/main-integration.yaml#L20)            | Waits for the operator's image build to finish so that image-dependent jobs can be executed.                                                       |
```